docs/LSPApi.md

```diff
@@ -26,6 +26,7 @@ The [LSPClientFeatures](https://github.com/redhat-developer/lsp4ij/blob/main/src
 - [LSP typeDefinition feature](#lsp-typeDefinition-feature)
 - [LSP usage feature](#lsp-usage-feature)
 - [LSP workspace symbol feature](#lsp-workspace-symbol-feature)
+- [LSP editor behavior feature](#lsp-editor-behavior-feature)

```

Comment:
Added documentation for the client-side editor behavior features exposed through client configuration now.